varlink/tests/test_error.py

```diff
@@ -0,0 +1,34 @@
+import json
+import unittest
+
+import varlink
+
+
+class OneMessageClientHandler(varlink.ClientInterfaceHandler):
+    def __init__(self, interface, namespaced, next_message):
+        # No interface but we do not use them
+        super().__init__(interface, namespaced)
+        self.next_message = next_message
+
+    def _next_message(self):
+        yield self.next_message
+
+
+class TestError(unittest.TestCase):
+    def test_pack_unpack(self):
+        dummy_if = varlink.Interface("interface org.example.dummy")
+        for error in [
+            varlink.InterfaceNotFound("org.varlink.notfound"),
+            varlink.MethodNotFound("Method"),
+            varlink.MethodNotImplemented("Abstract"),
+            varlink.InvalidParameter("Struct.param"),
```

Comment:
Noticed this doesn't differentiate for namespaced, while that is an accepted argument in `VarlinkError.__init__`.

Now that isn't something which will happen in practice but its a bit confusing. Also note VarlinkError has `parameters()` which does have namespaced support.

Reply:
Well the goal here was to test the client handling of those errors, not the Errors serialization which is supposed to be the same whether the error is namespaced or not (although that could be an interesting test as well).